packages/app/src/systems/DApp/pages/TxApprove/TxApprove.tsx

```diff
@@ -32,7 +32,7 @@ export function TxApprove({ id, url }: TxApproveProps) {
     <Layout title="New Transaction" isLoading={isLoading}>
       <Layout.TopBar type={TopBarType.external} />
       <Layout.Content css={styles.content}>
-        <UnlockDialog isOpen={Boolean(isLocked)} isFullscreen />
+        <UnlockDialog isFullscreen onUnlock={() => {}} isOpen={false} />
```

Comment:
Maybe remove completely the `onUnlock` if is not needed

Reply:
onUnlock is needed, but will be implemented in `txApprove` in next PR:

> `txApproval is gonna be broken until unlockMachine code changes are applied(Ref https://github.com/FuelLabs/fuels-wallet/issues/158 and https://github.com/FuelLabs/fuels-wallet/issues/159), which is gonna be my immediate next PR.`